code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -174,6 +174,23 @@
 	empty_contents()
 	. = ..()
 
+/mob/living/simple_animal/hostile/space_dragon/ex_act(severity, target, origin)
+	set waitfor = FALSE
+	if(origin && istype(origin, /datum/spacevine_mutation) && isvineimmune(src))
+		return
+	// Deal with parent operations
+	contents_explosion(severity, target)
+	SEND_SIGNAL(src, COMSIG_ATOM_EX_ACT, severity, target)
+	// Run bomb armour
+	var/bomb_armor = (100 - getarmor(null, BOMB)) / 100
+	switch (severity)
+		if (EXPLODE_DEVASTATE)
+			adjustBruteLoss(180 * bomb_armor)
```

Comment:
```suggestion
			adjustBruteLoss(250 * bomb_armor)
```

Reply:
This is only a suggested change for input from other people. This would leave them with 100 health, but maxcapping would kill them if they have under 100 health.

Reply:
Maxcap should absolutely kill space dragon. If they are that immune to explosions it would be absurd. The effort required to prepare and properly detonate a maxcap should absolutely be enough to kill a space dragon. They are threats, but not so powerful they can resist a maxcap.